invenio_oauthclient/contrib/github.py

```diff
@@ -161,7 +167,22 @@ def _extract_email(gh):
     return next((x.email for x in gh.emails() if x.verified and x.primary), None)
 
 
-def account_info(remote, resp):
+def default_info_serializer(remote, resp, user_info, github_obj):
+    """Serialize the account info response object."""
+    return dict(
+        user=dict(
+            email=_extract_email(github_obj),
+            profile=dict(
+                username=user_info.login,
+                full_name=user_info.name,
+            ),
+        ),
+        external_id=str(user_info.id),
+        external_method="github",
+    )
+
+
+def account_info(remote, resp, info_serializer=default_info_serializer):
```

Comment:
I also have some concerns on this one, since e.g. the `default_info_serializer` in this case also accepts the extra `github_obj` parameter.

If it's for modularity I would say it's ok, but in that case I would make it explicit also that the "signature" accepts `**kwargs`.

Reply:
oh, i think the "change" in my comment was ambiguous; i'm not proposing a change of the function
rather i'm just wondering if it's possible for users to override which `info_serializer` is being used by the function – if there is a place where users could pass a different argument, or if the function call is out of our hands (e.g. because it's called by a library?)

the splitting up of fetching and serializing logic makes total sense to me :+1:

Reply:
Ok, I thought I remembered this from somewhere, now I see where from :)

In that case, since it's just a convention within each contrib what parameters are being passed to their `info_serializer` I would leave it as is.

Reply:
Changes applied in a new commit, if you want to see the `diff`.